lib/parsers/index.ts

```diff
@@ -244,13 +244,28 @@ export function getTargetFrameworksFromProjectFile(manifestFile) {
     return targetFrameworksResult;
   }
   const propertyList = projectPropertyGroup
-    .find((propertyGroup) => _.has(propertyGroup, 'TargetFramework'));
+    .find((propertyGroup) => {
+        return _.has(propertyGroup, 'TargetFramework')
+        || _.has(propertyGroup, 'TargetFrameworks')
+        || _.has(propertyGroup, 'TargetFrameworkVersion');
+      }) || {};
 
-  if (!propertyList) {
+  if (_.isEmpty(propertyList)) {
     return targetFrameworksResult;
   }
-  for (const targetFramework of propertyList.TargetFramework) {
-    targetFrameworksResult = [...targetFrameworksResult, ...targetFramework.split(';')];
+  // tslint:disable
+  if (propertyList.TargetFrameworks) {
+    for (const item of propertyList.TargetFrameworks) {
+      targetFrameworksResult = [...targetFrameworksResult, ...item.split(';')];
+    }
+  }
+
+  if (propertyList.TargetFrameworkVersion) {
+    targetFrameworksResult = [...targetFrameworksResult, ...propertyList.TargetFrameworkVersion];
```

Comment:
Added lodash unique before return, it should NOT be possible to get repeats because users should not be using all 3 of these in 1 file but nothing will stop them from doing so.